oak-search-elastic/pom.xml

```diff
@@ -33,8 +33,8 @@
   <description>Oak Elasticsearch integration subproject</description>
 
   <properties>
-    <elasticsearch.version>7.16.3</elasticsearch.version>
-    <lucene.version>8.10.1</lucene.version>
+    <elasticsearch.version>7.17.3</elasticsearch.version>
+    <jackson.version>2.12.3</jackson.version>
```

Comment:
If possible can we keep this in sync with the jackson dependency in the parent pom - https://github.com/apache/jackrabbit-oak/blob/trunk/oak-parent/pom.xml#L66. ? Either downgrade this or upgrade that.